backend/tests/test_auth.py

```
@@ -34,26 +33,35 @@
-        db.refresh(user)
-    db.close()
+        db_session.add(user)
+        db_session.commit()
+        db_session.refresh(user)
     return user
 
+# ----------------------
+# 🔧 Fixture for DB session (replaces direct SessionLocal usage)
+# ----------------------
+@pytest.fixture
+def db_session():
```

Ruff (0.8.2)

Missing return type annotation for public function `db_session`

(ANN201)